### PR TITLE
add additional SSLHandshakeException

### PR DESCRIPTION
**Pull request by geekpete**
Add addition to SSLHandshakeException troubleshooting guide for ECDH ServerKeyExchange
that might be caused by cert/cacert/key mismatch.

### CODE REVIEW

File: docs/en/stack/security/troubleshooting.asciidoc [hunk 1]
```diff
@@ -244,6 +244,14 @@ found in the `keystore` or `truststore` and needs to be added to trust this
 certificate.
 --
 
+`javax.net.ssl.SSLHandshakeException: Invalid ECDH ServerKeyExchange signature``::
++
+--
+The `Invalid ECDH ServerKeyExchange signature` can indicate that a key and a corresponding certificate don't match and are
+causing the handshake to fail.
+Verify that any configured certificate, CA certificate and key are the correct ones and are matching.
```
Comment (tvernum) [inline docs/en/stack/security/troubleshooting.asciidoc]:
```suggestion
Verify the contents of each of the files you are using for your configured certificate authorities (CAs), certificates and keys. In particular, check that the key and certificate belong to the same key pair. 
```

**Comment by nik9000** [comment]
@elasticmachine, run docs build

**Comment by nik9000** [comment]
@elasticmachine, run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine, retest this please

**Comment by nik9000** [comment]
@elasticmachine, run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by nik9000** [comment]
@elasticmachine run docs build

**Comment by geekpete** [comment]
@elasticmachine run docs build

**Comment by tvernum** [comment]
@geekpete Would you like me to handle the merge & backport for this?

**Comment by tvernum** [comment]
@elasticmachine update branch

**Pull request merged**